### PR TITLE
SSL configuration creation

### PR DESCRIPTION
**Pull request by Pe46dro**
PR as requested #28 

Now the file generation will be made after domain variable definition and configuration will be tested with the correct name

Sorry for multiple commits but i made it from Github Web

### CODE REVIEW

**Comment by enoch85** [comment]
@ezraholm50 Please test this, doesn't seem right at first look. Thanks!

@Pe46dro Thanks for your PR!!

File: lets-encrypt/activate-ssl.sh [hunk 1]
```diff
@@ -242,7 +245,7 @@ fi
 # Check if $certfiles exists
 if [ -d "$certfiles" ]; then
 # Activate new config
-        bash /var/scripts/test-new-config.sh
+        bash /var/scripts/test-new-config.sh $domain.conf
```
Comment (ezraholm50) [inline lets-encrypt/activate-ssl.sh]: Have you actually tested this?

Comment (Pe46dro) [inline lets-encrypt/activate-ssl.sh]: Proof of work: http://imgur.com/a/sDALF

I made two gist with the script so you can test it.
https://gist.github.com/Pe46dro/a6e26ce6c3bc6d0a70ff8053d80b5a3c
https://gist.github.com/Pe46dro/c9c1e84c1778f733d54d5954bb7cdd81

Comment (enoch85) [inline lets-encrypt/activate-ssl.sh]: `test-new-config.sh` is a totally different script that tests the configuration and has nothing to do with the domain name. @Pe46dro Please explain this fix.

**Comment by enoch85** [comment]
I just checked this a little but more and understands the fix, not tested though. 

Lets's merge this anyway and see what response we get from other users. 

👍 From me.

**Comment by Pe46dro** [comment]
Essentially before my change when you run the `lets-encrypt/activate-ssl.sh` file a `.conf` file was generated (only `.conf`, no name, because the `$domain` used in `$ssl_conf` definition was defined before FQDN input ) and when you run the `test-new-config.sh` you always try to enable the `nextcloud_ssl_domain.conf` configuration.

**Pull request merged**